src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -165,7 +165,11 @@ public function get_registered( $pattern_name ) {
 			return null;
 		}
 
-		return $this->registered_patterns[ $pattern_name ];
+		$pattern            = $this->registered_patterns[ $pattern_name ];
+		$blocks             = parse_blocks( $pattern['content'] );
+		$visitor            = _parsed_block_visitor( $pattern ); // TODO: Should we use different functions for template vs pattern?
+		$pattern['content'] = serialize_blocks( $blocks, $visitor );
```

Comment:
TODO: We probably want some kind of filter here to allow modifying the pattern content.